src/css/stylesheet.css

```diff
@@ -13,6 +13,7 @@
 }
 
 #home {
+    background-image: url("/img/bg.png");
```

Comment:
Why do we need this? This is already in the bg class

Reply:
well something went wrong during the rebase. I didn't introduce it deliberately

Reply:
You shall not pass!